Add and test training in lite interpreter.

iseeyuan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32359 Add and test training in lite interpreter.**

Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

Commit:
Add and test training in lite interpreter.

Commit:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

Commit:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

Commit:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 22364694:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=223646944bdfca69fff8d86d8d2d813a3108cacc).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4307234) pytorch_linux_xenial_py3_clang5_mobile_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=88978653) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=88978653))
<details>
<summary>
<code>Jan 21 20:15:46 caused by: Connection refused (os error 111)</code>
</summary>

```
Jan 21 20:15:46 +++ eval 'extract_trap_cmd ' 
Jan 21 20:15:46 ++++ extract_trap_cmd 
Jan 21 20:15:46 ++++ printf '%s\n' '' 
Jan 21 20:15:46 +++ printf '%s\n' cleanup 
Jan 21 20:15:46 ++ trap -- ' 
Jan 21 20:15:46 cleanup' EXIT 
Jan 21 20:15:46 ++ which sccache 
Jan 21 20:15:46 ++ sccache --stop-server 
Jan 21 20:15:46 Stopping sccache server... 
Jan 21 20:15:46 error: couldn't connect to server 
Jan 21 20:15:46 caused by: Connection refused (os error 111) 
Jan 21 20:15:46 ++ true 
Jan 21 20:15:46 ++ rm /var/lib/jenkins/sccache_error.log 
Jan 21 20:15:46 rm: cannot remove '/var/lib/jenkins/sccache_error.log': No such file or directory 
Jan 21 20:15:46 ++ true 
Jan 21 20:15:46 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Jan 21 20:15:46 ++ SCCACHE_IDLE_TIMEOUT=1200 
Jan 21 20:15:46 ++ RUST_LOG=sccache::server=error 
Jan 21 20:15:46 ++ sccache --start-server 
Jan 21 20:15:46 Starting sccache server... 
Jan 21 20:15:46 ++ sccache --zero-stats 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

iseeyuan committed:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

[ghstack-poisoned]

iseeyuan committed:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

[ghstack-poisoned]

iseeyuan committed:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

[ghstack-poisoned]

iseeyuan committed:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

[ghstack-poisoned]

Commit:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

Commit:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

Commit:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

Commit:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

Commit:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)

Commit:
Update on "Add and test training in lite interpreter."


Differential Revision: [D19450614](https://our.internmc.facebook.com/intern/diff/D19450614)
